tests/manual/controls/SelectListViewTest.java

```diff
@@ -0,0 +1,96 @@
+import javafx.application.Application;
+import javafx.collections.ObservableList;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.ListView;
+import javafx.scene.control.SelectionMode;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+public class SelectListViewTest extends Application {
+
+    final int ROW_COUNT = 70_000;
+    //  final int ROW_COUNT = 400_000;
+    //  final int ROW_COUNT = 400_000;
+    //  final int ROW_COUNT = 10_000_000;
+    // final int ROW_COUNT = 7_000;
+
```

Comment:
There is a duplication of the number of test data.

My test is below.

``` Java
public class SelectListViewTest extends Application {
	final int ROW_COUNT = 70_000;
//	final int ROW_COUNT = 400_000;
//	final int ROW_COUNT = 10_000_000;
//	final int ROW_COUNT = 7_000;


```

Reply:
Thanks. I have fixed this.